HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt

```diff
@@ -0,0 +1,239 @@
+package com.chartboost.heliumsdk.sampleapp
+
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.text.method.ScrollingMovementMethod
+import androidx.appcompat.app.AppCompatActivity
+import com.chartboost.heliumsdk.HeliumSdk
+import com.chartboost.heliumsdk.ad.*
+import com.chartboost.heliumsdk.sampleapp.databinding.ActivityMainBinding
+import java.util.*
+
+class MainActivity : AppCompatActivity() {
+    private var _bannerAd: HeliumBannerAd? = null
+    private lateinit var binding: ActivityMainBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+        setupLogView()
+        setupScreen()
+        setupInterstitial()
+        setupRewarded()
+        setupListeners()
+        setupBanner()
+        setupSdk()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        _bannerAd?.destroy()
```

Comment:
We should be destroying all placements we are using, right? Otherwise we'll leak the activity.

Reply:
We have just one placement , the one in _bannerAd, what else do we have to destroy?

Reply:
Think @bichenwang means the other samples for interstitial and rewarded too.

Reply:
sure